File: ISL.ReIdentification.WebApps.Configurations/ISL.ReIdentification.Configurations.Server.Tests.Acceptance/Models/PdsDatas/PdsData.cs
// ---------------------------------------------------------
// Copyright (c) North East London ICB. All rights reserved.
// ---------------------------------------------------------

using System;

namespace ISL.ReIdentification.Configurations.Server.Tests.Acceptance.Models.PdsDatas
{
    public class PdsData
    {
        public Guid Id { get; set; }
        public string PseudoNhsNumber { get; set; }
        public string OrgCode { get; set; }
        public string OrganisationName { get; set; }
        public DateTimeOffset? RelationshipWithOrganisationEffectiveFromDate { get; set; }
        public DateTimeOffset? RelationshipWithOrganisationEffectiveToDate { get; set; }
    }
}

File: ISL.ReIdentification.WebApps.Configurations/ISL.ReIdentification.Configurations.Client/src/models/pds/pdsData.ts

export class PdsData {
    public id: string;
    public pseudoNhsNumber: string;
    public orgCode: string;
    public organisationName: string;
    public relationshipWithOrganisationEffectiveFromDate?: Date | undefined;
    public relationshipWithOrganisationEffectiveToDate?: Date | undefined;

    constructor(pds: PdsData) {
            this.id = pds.id;
            this.pseudoNhsNumber = pds.pseudoNhsNumber;
            this.orgCode = pds.orgCode;
            this.organisationName = pds.organisationName
            this.relationshipWithOrganisationEffectiveFromDate = pds.relationshipWithOrganisationEffectiveFromDate
            this.relationshipWithOrganisationEffectiveToDate = pds.relationshipWithOrganisationEffectiveToDate;
    }
}